src/change_list.rs

```diff
@@ -267,6 +267,24 @@ enum ChangeDiscriminant {
     /// dropString(id);
     /// ```
     DropString = 15,
+
+    /// Immediates: `(pointer, length)`
+    ///
+    /// Stack: `[...] -> [... Node]`
+    ///
+    /// ```text
+    /// stack.push(createElement(String(pointer, length))
+    /// ```
+    CreateElementNS = 16,
+
+    /// Immediates: `(pointer, length)`
```

Comment:
This has `(pointer1, length1, pointer2, length2)` immediates, right? Same thing for the other new instruction above.

Reply:
Yea, the docs were just copy pasta for now. Will change.